README.md

```diff
@@ -59,7 +59,7 @@ Ensure the properties there fit your environment.
 Simply copy the **menas.war** file produced when building the project into Tomcat's webapps directory. 
 
 #### Standardization and Conformance requirements:
-- **Spark 2.4.0** installation
+- **Spark 2.4.1** installation
```

Comment:
Should we change this to `2.4.1+`or `2.4.3`?

Reply:
I think 2.4.x should be compatible with each other. Unless we've noticed a major bug on one of the incremental versions.

Reply:
There was a big in 2.4.0 with `distinct` that was fixed in 2.4.1 and some incompatibilities which prompted the upgrade of `spark-hofs`

Reply:
Let me update the docs real quick...

Reply:
Made it Spark 2.4.3 (Scala 2.11) since spark-hofs uses 2.4.3 as well. Although this should work on 2.4.1+, I'd prefer to be specific in README for now.